pkgs/applications/audio/cmusfm/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, curl, libnotify, gdk_pixbuf }:
+
+stdenv.mkDerivation rec {
+  version = "ad2fd0aad3f4f1a25add1b8c2f179e8859885873";
```

Comment:
You should use `cmusfm-unstable-yyyy-mm-dd`, see the manual section on [package naming](https://nixos.org/nixpkgs/manual/#sec-package-naming).

Reply:
And your commit message corrected accordingly ( `cmusfm: init at 2018-10-11` )